addons/survey/static/src/scss/survey_templates_results.scss

```diff
@@ -12,10 +12,47 @@
     html {
         height: unset;
     }
+    @page {
+        size: portrait; /* force paper orientation to portrait */
+    }
+    canvas {
```

Comment:
'survey_templates_results.scss' seems included in 'web.assets_backend', aren't we going to modify the behavior of all canvas here ? Shouldn't we namespace to be sure this css applies only to survey result printing ?

Reply:
As you suggested, I will embed style rules within .o_survey_result to limit their scope to survey result page :+1: